docs/BuildFromSource.md

```diff
@@ -15,13 +15,13 @@ This tutorial assumes that you are familiar with:
 
 ASP.NET Core uses git submodules to include the source from a few other projects. In order to pull the sources of the these submodules when cloning the repo, be sure to pass the `--recursive` flag to the `git clone` command.
 
-```ps1
+```powershell
```

Comment:
This file wasn't really incredibly broken; it just gave the VS Code Markdown extension plenty to complain about